### DISCUSSION TITLE
how do i disable tablet pen tip click?

### DISCUSSION THREAD

**Discussion by kenraali** [ppy/osu#26317] [Q&A] [answered]
i have tried to find answer to this for last 1 hour, and i cant find any.

**Comment by Rekunan** [comment]
Currently, your best bet is to disable the ingame drivers and use an external driver like [OTD](https://opentabletdriver.net/) to do so.

**Comment by Susko3** [comment, marked as answer]
Tablet clicks are controller by the same setting that controls mouse button clicks. So you can either disable mouse button clicks during gameplay, or unmap the left mouse button (`M1`) from osu! ruleset keybinds.

This only applies to gameplay. Tablet pen clicks will always work as the left mouse click on other screens (eg. main menu, song select).
